docs/fa/docs/css/rtl.css

```diff
@@ -0,0 +1,15 @@
+.termy{
+    direction: ltr;
+}
+code{
+    direction: ltr;
+    display: inline-block;
+}
+.external-link{
+    direction: ltr;
+    display: inline-block;
+}
+
+.md-content__inner h1{
+    direction: ltr !important;
+}
```

Comment:
As I see that most of these settings only set some elements to always be `ltr`, I think it would be safe to move them to the main CSS (at `docs/en/docs/css/custom.css`). That file is then copied to all the languages.

Also because when the MkDocs configs are regenerated the custom CSS for Persian would be overridden, so, if these CSS styles can be put in a way that applies to everything, it will all work correctly. But if it's absolutely needed to have custom CSS styles only for right-to-left languages, then I would need to figure out a way to support it in the scripts that generate the multi-language docs. 😬

But that would also depend on if the `display: inline-block;` are necessary or not...

Reply:
Yes, the changes that make some styles `ltr` are safe in every other language and it makes sense because they are applied to codes and links and these two items are always ltr, no matter what the language is. But I will pull the latest changes and check if everything is OK before resolving this thread.

Reply:
I checked it. Yes it is totally safe to move these `ltr` changes to the main CSS.

Reply:
I tried to check the latest changes and then reverted my branch to the final changes you made.